tests/test_misclassifications.py

```diff
@@ -0,0 +1,72 @@
+import pandas as pd
+import pytest
+
+from presc.misclassifications.misclass_rate import (
+    misclass_rate_feature,
+    show_misclass_rate_feature,
+    show_misclass_rates_features,
+    compute_quantiles,
+    show_quantiles_feature,
+    show_quantiles_features,
+)
+
+
+@pytest.fixture
+def test_dataset():
```

Comment:
This is kind of a nit, but convention is that functions that run tests are named `test_....`, so fixtures shouldn't be. You can just call this `dataset`.